src/core/permissions/utils/computePermissionAssignments.js

```diff
@@ -26,17 +26,20 @@ function computePermissionAssignments(
 ) :{
   isAssignedToAll :boolean;
   isAssignedToOnlyNonPII :boolean;
+  isOwnerOnAtLeastOneColumn :boolean;
 } {
 
   let isAssignedToAll = true;
   let isAssignedToOnlyNonPII = true;
+  const ownershipByColumn = [];
 
   dataSetColumns.forEach((column :Map<FQN, List>) => {
     const columnId :UUID = getPropertyValue(column, [FQNS.OL_ID, 0]);
     const propertyType :?PropertyType = maybePropertyTypes.get(columnId);
     const pii :boolean = propertyType?.pii || false;
     const key :List<UUID> = List([dataSetId, columnId]);
     const isOwner = myKeys.has(key);
+    ownershipByColumn.push(isOwner);
```

Comment:
Using an array seems overkill since all you need to know is if there exists one case where `isOwner === true`. I think the following will work (and is slightly more efficient):

```
let isOwnerOfAtLeastOneColumn = false;
...
const isOwner = myKeys.has(key);
if (isOwner) {
  isOwnerOfAtLeastOneColumn = true;
  ...
}
```